<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassPropertiesBase</class>
 <widget class="QDialog" name="ClassPropertiesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Class Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Python base classes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pyBaseClasses"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Annotations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="abstractCb">
        <property name="text">
         <string>Abstract</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="noqmetaobjectCb">
        <property name="text">
         <string>PyQtNoQMetaObject</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="allownoneCb">
        <property name="text">
         <string>AllowNone</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="8">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>PyQtInterface</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="interface"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Supertype</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="supertype"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>TypeHint</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="typeHint"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>TypeHintIn</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="typeHintIn"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>TypeHintOut</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="typeHintOut"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>TypeHintValue</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="typeHintValue"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>API</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="api"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="delaydtorCb">
        <property name="text">
         <string>DelayDtor</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="exportderivedCb">
        <property name="text">
         <string>ExportDerived</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Metatype</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="metatype"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="mixinCb">
        <property name="text">
         <string>Mixin</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="nodefaultctorsCb">
        <property name="text">
         <string>NoDefaultCtors</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="noTypeHintCb">
        <property name="text">
         <string>NoTypeHint</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>PyName</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="pyName"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pyBaseClasses</tabstop>
  <tabstop>abstractCb</tabstop>
  <tabstop>allownoneCb</tabstop>
  <tabstop>api</tabstop>
  <tabstop>delaydtorCb</tabstop>
  <tabstop>exportderivedCb</tabstop>
  <tabstop>metatype</tabstop>
  <tabstop>mixinCb</tabstop>
  <tabstop>nodefaultctorsCb</tabstop>
  <tabstop>noTypeHintCb</tabstop>
  <tabstop>pyName</tabstop>
  <tabstop>noqmetaobjectCb</tabstop>
  <tabstop>interface</tabstop>
  <tabstop>supertype</tabstop>
  <tabstop>typeHint</tabstop>
  <tabstop>typeHintIn</tabstop>
  <tabstop>typeHintOut</tabstop>
  <tabstop>typeHintValue</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClassPropertiesBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClassPropertiesBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
